go/vt/vttablet/tabletserver/vstreamer/resultstreamer.go

```diff
@@ -97,6 +99,7 @@ func (rs *resultStreamer) Stream() error {
 
 	response := &binlogdatapb.VStreamResultsResponse{}
 	byteCount := 0
+	logger := logutil.NewThrottledLogger(rs.vse.GetTabletInfo(), throttledLoggerInterval)
```

Comment:
Any reason not to use something like:
```
logger := logutil.NewThrottledLogger(rs.vse.GetTabletInfo()+":rowstreamer:"+rs.plan.Table.Name, throttledLoggerInterval)
```

I didn't see a way to get at the workflow name as well.